ci-scripts/dependencies.sh

```diff
@@ -17,6 +17,7 @@ fi
 PATH="~/.yarn/bin:$PATH"
 
 yarn
+cd test/integration && yarn && cd ../..
```

Comment:
The root `package.json` has [this dependency](https://github.com/mapbox/mapbox-gl-js/blob/af7ae75c8ea4a34272c9c3651bcbfd3b42cf1615/package.json#L50) which triggers installation of the integration dependencies automatically. Can you adjust the path munging for `local://` URLs so that they load from the root `node_modules`, making this line unnecessary?